lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -476,6 +476,42 @@ describe('AlfrescoViewerComponent', () => {
         //
     });
 
+    describe('originalMimeType', () => {
+        it('should set originalMimeType based on nodeData content', async () => {
+            const defaultNode: Node = {
+                id: '123',
+                name: 'Mock Node',
+                nodeType: 'cm:content',
+                isFolder: false,
+                isFile: true,
+                modifiedAt: new Date(),
+                modifiedByUser: { id: 'user123', displayName: 'John Doe' },
+                createdAt: new Date(),
+                createdByUser: { id: 'user456', displayName: 'Jane Doe' },
+                isLocked: false,
+                parentId: 'parent123',
+                isLink: false,
+                isFavorite: false,
+                content: {
+                    mimeType: 'application/msWord',
+                    mimeTypeName: 'Doc Document',
+                    sizeInBytes: 0
+                },
+                aspectNames: ['cm:auditable', 'cm:versionable'],
+                properties: { customProperty: 'customValue' },
+                allowableOperations: ['update', 'delete', 'create-child'],
+                path: { name: '/Sites/Document Library/Mock Node', isComplete: true },
+                permissions: {},
+                definition: {}
+            };
+
+            spyOn(component['viewUtilService'], 'getViewerType').and.returnValue('unknown');
```

Comment:
You should not refer to service in that way (by using []). It is workaround which we should not use. It's not proper way to get instance of private field. 
Instead you can use viewUtilService variable which is already assigned in tests in 173rd line. It is assigned there by TestBed.inject so just refer to viewUtilService variable which already exists instead of reading it from component.

<img width="891" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/64f8293d-3bcc-4e8c-a396-fc2c6d9cb466">